packages/client/hmi-client/src/components/widgets/vue-show-more-text.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+	<div class="container" v-bind:style="additionalContainerCss">
+		<!-- eslint-disable -->
+		<div
+			ref="detail"
+			v-bind:class="{ default: !expanded, expanded: expanded }"
+			v-bind:style="`--lines: ${lines}`"
+		>
+			<div v-bind:style="additionalCss" v-html="text"></div>
+		</div>
+		<!-- eslint-enable -->
+		<a
+			v-if="hasMore && triggerShowMore"
+			class="anchor"
+			:style="additionalAnchorCss"
+			@click="onClick"
+			>{{ moreText }}</a
+		>
+		<a v-if="hasMore && expanded" class="anchor" :style="additionalAnchorCss" @click="onClick">{{
+			lessText
+		}}</a>
+	</div>
+</template>
+
+<script lang="ts">
```

Comment:
Could you rewrite this component in script-setup using Vue3 features.
You can also rename it to `<tera-show-more-text>` to follow our new guidelines.

Reply:
We can also remove all the props we do not use.
Let make this component our own completely.